docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md

```diff
@@ -17,8 +17,8 @@ management workspace (WMW).  Start by creating a WMW for the common
 workload, with the following commands.
 
 ```shell
-IN_CLUSTER=false SPACE_MANAGER_KUBECONFIG=$SM_CONFIG kubectl kubestellar ensure wmw wmw-c
-wmw_c_space_config=$PWD/temp-space-config/spaceprovider-default-wmw-c
+IN_CLUSTER=false kubectl kubestellar ensure wmw wmw-c
```

Comment:
What is the default behavior of `kubectl kubestellar ensure wmw` if the `IN_CLUSTER` envar is not set? I think there should be a default, and it should be the safe choice --- the one that works even if less efficiently than possible.